src/Serialization.NewtonsoftJson/Serialization.NewtonsoftJson.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>GraphQL.Server.Serialization.NewtonsoftJson</AssemblyName>
+    <RootNamespace>GraphQL.Server.Serialization.NewtonsoftJson</RootNamespace>
+    <PackageId>GraphQL.Server.Serialization.NewtonsoftJson</PackageId>
+    <Description>GraphQL middleware using Newtonsoft.Json</Description>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Transports.AspNetCore\Transports.AspNetCore.csproj" />
```

Comment:
I would like to move all classes not related to AspNetCore (`GraphQLRequest`, `GraphQLRequestExecutionResult`) from Transports.AspNetCore project to Core project and change dependency to Core instead of Transports.AspNetCore. Also for GraphQL.Server.Serialization.SystemTextJson. Serialization is transport agnostic thing.

Also you may remove `  <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="$(MicrosoftExtensionsDependencyInjectionAbstractionsVersion)" />` from Core since Microsoft.Extensions.Options already contains it.